source/_integrations/letpot.markdown

```diff
@@ -56,6 +58,26 @@ Password:
 
 The LetPot integration provides the following entities.
 
+#### Binary sensors
+
+- **Pump**: Indicates if the water pump is running (on) or idle (off). Updates may be delayed by the device until another entity updates.
+  - Available for LetPot Air devices, and other device models which report this value, but never on LetPot Pro.
+
+Additionally, binary sensors for possible issues are available:
+
+- **Low nutrients**: Indicates if the nutrient solution tank used for auto mode is (almost) empty and should be refilled.
+  - Available for LetPot Max devices.
+- **Low water**: Indicates if the water tank is (almost) empty and should be refilled.
+  - Available for all devices except LetPot Pro.
+- **Pump error**: Indicates if the water pump detected a flow issue and should be cleaned.
+  - Available for some LetPot Mini and LetPot SE devices which report this value.
+- **Refill error**: Indicates if the external water tank/input used for refilling in auto mode is not increasing the main water tank level and should be checked.
```

Comment:
```suggestion
- **Refill error**: Indicates that the external water tank/input used for refilling in auto mode is not filling up the main water tank. If you see a refill error, check the connection to the water source.
```

Reply:
Thanks, not putting everything in one sentence helps :)

I've changed the suggestion because there can be 2 causes, the water tank/input itself and the connection to the main water tank.